src/gocept/runner/transaction.py

```diff
@@ -1,11 +1,12 @@
 # Copyright (c) 2011 gocept gmbh & co. kg
 # See also LICENSE.txt
-
 from __future__ import absolute_import
-import ZODB.POSException
-import decorator
+
 import logging
```

Comment:
Echt, muss das so? Verrückt.

Reply:
Ich habe jetzt einen sorter installiert. Er macht, was er soll. Nun haben wir es hier dann doch auch in drei Abschnitten.